binaries.yaml

```diff
@@ -16,6 +16,9 @@ oses:
     zip: '"C:\Program Files\7-Zip\7z.exe" a -tzip'
 
 ghcs:
+  ghc-9.8.4:
```

Comment:
Are there other `ghc-9.8.x`s we should add?

Reply:
Would it need to be in Stackage? Because the LTS series skips right from 9.6.6 to 9.8.4